cypress/fixtures/analysis.json

```diff
@@ -244,5 +244,26 @@
     "source_analysis_on_pet_clinic_app": {
         "source": "Source code",
         "appName": "pet-clinic"
+    },
+
+    "source+dep_analysis_on_daytrader-app-affected-files": {
```

Comment:
We can give a shorter name like : 
"affected_files_on_day_trader_app"

Reply:
Done, Thanks!